lib/core/public/configure.js

```diff
@@ -45,14 +45,34 @@ function configure(spec) {
 	}
 
 	if (spec.checks) {
+		if (!Array.isArray(spec.checks)) {
+			throw new Error('Checks property must be an array');
```

Comment:
```suggestion
			throw new TypeError('Checks property must be an array');
```

Reply:
Good idea. Will fail the build after i merge so ill fix it with another commit once I merge these in.